frontend/src/api/k8s/pipelineRuns.ts

```diff
@@ -0,0 +1,27 @@
+import { k8sListResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { PipelineRunKind } from '~/k8sTypes';
+import { PipelineRunModel } from '~/api/models';
+
+export const listK8sPipelineRunsByLabel = async (
+  namespace: string,
+  label: string,
+): Promise<PipelineRunKind[]> =>
+  k8sListResource<PipelineRunKind>({
+    model: PipelineRunModel,
+    queryOptions: {
+      ns: namespace,
+      queryParams: { labelSelector: label },
+    },
+  }).then((listResource) => listResource.items);
+
+export const getK8sPipelineRun = async (
```

Comment:
Isn't this should be `listK8sPipelineRun`? 

Should I change this in my working PR?

Reply:
yeah go ahead. i wasnt using it . i just made it for an example